sentinelsat/sentinel.py

```diff
@@ -289,6 +290,52 @@ def get_footprints(self, products):
             )
         return geojson.FeatureCollection(feature_list)
 
+    def to_dict(self, products):
+        """Return the products in a dictionary object with the values in their appropriate Python types"""
+        try:
+            from osgeo import ogr
```

Comment:
GDAL/OGR is one hell of a dependency for a tool thats just supposed to be a search/download helper. Since we are in the process of moving away from compiled extensions (see #91) I am strongly against including new compiled dependencies, especially such complex ones - unless they provide a large benefit to `sentinelsat` as a whole.